index.html

```diff
@@ -898,14 +898,166 @@ <h4 class="metrics-label">Dana Scully</h4>
                 </div>
             </li>
         </ul>
-        
+
         <style type="text/css">
             /* Funtivity CSS goes here */
+            .stat-table {
+                display: table;
+                color: #f46240;
+                width: 100%;
+            }
+            .stat-table > div {
+                display: table-cell;
+            }
+            .stat-table .info {
+                width: 60%;
+                font-weight: bold;
+            }
+            .stat-table .number {
+                font-size: 32px;
+                line-height: 34px;
+            }
+            .stat-table .label {
+                text-transform: uppercase;
+                font-size: 12px;
+            }
+            .stat-table .ss-icon {
+                font-size: 36px;
+                line-height: 36px;
+                vertical-align: middle;
+                text-align: center;
+                width: 40%;
+            }
+
```

Comment:
I'm not 100% sure I agree with that, but let's talk about it. To me this feels like overengineering too early - remember the steps are to add it page-by-page, then if there are multiples, you can split off. We don't know yet whether there will EVER be another mini stats table, and we don't know yet what it will look like if there is. So that's why I think it's a good idea to always introduce new styles as a one-off, and then port them over when it makes sense.